src/mod/endpoints/mod_sofia/sofia_glue.c

```diff
@@ -1475,7 +1475,7 @@ switch_status_t sofia_glue_do_invite(switch_core_session_t *session)
 			switch_channel_set_variable(channel, "sip_nat_detected", "true");
 		}
 
-		if ((val = switch_channel_get_variable(channel, "sip_cid_type"))) {
+		if (!(tech_pvt->profile->force_cid_type) && (val = switch_channel_get_variable(channel, "sip_cid_type"))) {
```

Comment:
If force_cid_type is set what would be the value of cid_type here? If it extract from the profile then it may be CID_TYPE_NONE. Then this [line](https://github.com/team-telnyx/freeswitch/pull/307/files#diff-d5d43948c580fd7dc42f28628beb5ff25ee9c4cf09c2154996b807720ba96264R1511) may not be hit unless the profile cid_type is set to CID_TYPE_PID?

Reply:
We're going to include this change on b2bua v68

Reply:
This is resolved